workers/ohsome_quality_analyst/cli.py

```diff
@@ -223,12 +215,9 @@ def create_report(
         if outfile:
             if fid_field is None:
                 fid_field = DATASETS[dataset_name]["default"]
-            feature_collection = asyncio.run(
-                db_client.get_bpolys_from_db(dataset_name, feature_id, fid_field)
-            )
-            for feature in feature_collection.features:
-                feature = update_features_report(feature, report)
-            write_geojson(outfile, feature_collection)
+            feature = asyncio.run(db_client.get_region_from_db(feature_id, fid_field))
+            feature = update_features_report(feature, report)
+            write_geojson(outfile, feature)
```

Comment:
Before this PR `get_region_from_db()` returned a FeatureCollection with one Feature. I changed this to only return the Feature. 

If the parameter to the CLI are `dataset` and `feature_id` then OQT always act upon one feature only.
If on the other hand a GeoJSON is passed in then it can also be a FeatureCollection with multiple Features. See lines above at the beginning of the function (183).